pkg/config/config.go

```diff
@@ -3,14 +3,13 @@ package config
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/ghodss/yaml"
```

Comment:
does not look like `go fmt` formatted here. These changes break a proper formatting here ...

Reply:
`go fmt ./...` changed that on my side. When these lines are moved back to old location and format the code afterwards it gets moved back to the state like it is. Do we have different formatting behavious on different local environments? Would be bad ...